src/content/zh-CN/2022/pwa.md

```diff
@@ -0,0 +1,524 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: PWA
+description: 2022 Web 年鉴的 PWA 章节涵盖 Service Workers 的使用和功能、Web 应用程序清单、Lighthouse 的洞察、Service Worker 库（包括 Workbox）和 Web 推送通知。
+authors: [diekus]
+reviewers: [aarongustafson, tropicadri, webmaxru, Schweinepriester, beth-panx]
+analysts: [beth-panx]
+editors: [siwinlo, tunetheweb]
+translators: [ibelem]
+results: https://docs.google.com/spreadsheets/d/1PbzjhN%2D%2DjU9MGuWobw5L9EsmlVzI9tlbCe3_NKA7giU/
+diekus_bio: Diego Gonzalez 是来自哥斯达黎加的计算机工程师，在 Microsoft Edge 浏览器团队担任 PWA 平台功能的 PM。
+featured_quote: 随着可安装的 Web 应用与桌面平台集成的功能不断增加，推动了业内知名厂商使用 PWA。
+featured_stat_1: 95%
+featured_stat_label_1: 桌面端可 JSON 解析的 manifest 文件
+featured_stat_2: 63%
+featured_stat_label_2: 桌面端 PWA 使用 Service Worker 的 install 事件
+featured_stat_3: 60%
+featured_stat_label_3: 以某种方式使用 Workbox 的 PWA 应用
+---
+
+## 简介
+
+在渐进式 Web 应用 PWA 的早期，有两个关键功能吸纳了现代 Web 应用的能力，离线支持和添加到主屏幕的快捷方式。
+
+这两个概念在安装 PWA 后启用，该过程通常从点击浏览器地址栏中出现的安装图标开始。该图标会提示用户安装该网站。Samsung Internet 和 Mozilla Firefox 等移动浏览器最早明确支持此行为，被熟知为<a hreflang="zh-CN" href="https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps/Add_to_home_screen">`添加到主屏幕 (A2HS)`</a>。
```

Comment:
For MDN we will automatically not the language based on the URLs to no need to convert to HTML links. Also I don't think this should be in code format?

```suggestion
这两个概念在安装 PWA 后启用，该过程通常从点击浏览器地址栏中出现的安装图标开始。该图标会提示用户安装该网站。Samsung Internet 和 Mozilla Firefox 等移动浏览器最早明确支持此行为，被熟知为[添加到主屏幕 (A2HS)](https://developer.mozilla.org/zh-CN/docs/Web/Progressive_web_apps/Add_to_home_screen)。
```